src/core/Checkbox/index.tsx

```diff
@@ -1,10 +1,56 @@
-import { Checkbox as RawCheckbox, CheckboxProps } from "@material-ui/core";
+import { CheckboxProps as MUICheckboxProps, SvgIcon } from "@material-ui/core";
 import React from "react";
+import IconCheckboxChecked from "../../common/svgs/IconCheckboxChecked.svg";
+import IconCheckboxIndeterminate from "../../common/svgs/IconCheckboxIndeterminate.svg";
+import IconCheckboxUnchecked from "../../common/svgs/IconCheckboxUnchecked.svg";
+import { StyledCheckbox } from "./styles";
 
-export { CheckboxProps };
+interface CheckboxProps
```

Comment:
Let's export the interface too, in case people use it!

```suggestion
export interface CheckboxProps
```

Reply:
Woah! I've never seen this Github feature

Reply:
Yaas! Magical button 😆 

<img width="611" alt="Screen Shot 2021-08-16 at 11 02 14 AM" src="https://user-images.githubusercontent.com/6309723/129608925-7caea84b-5cf3-47d3-8e5c-103a08f1e597.png">